RELEASE.md

```diff
@@ -13,3 +13,12 @@ the process is as follows:
 1. Run the release command using `GH_TOKEN=<token> npm run release`
 1. If release is successful, push commits and tags upstream with
    `git push upstream main && git push upstream --tags`
+
+#### CI based
+
+The release process is also automated in the way any specific commit from the main branch can be potentially released, for such it's required the below steps:
+
+1. Be sure you have checked out the `main` branch and have pulled the latest changes
+1. Bump the alpha version by running `npm version prerelease --preid=alpha`
+1. Push commits and tags upstream with `git push upstream main && git push upstream --tags`
```

Comment:
If needed, it could be done automatically in the CI with an input parameter, for instance -> [apm-agent-rum-js](https://github.com/elastic/apm-agent-rum-js/blob/main/RELEASE.md#ci-based) uses an input approval approach and it's UI based.